src/messages.rs

```diff
@@ -34,6 +34,8 @@ pub struct ArtifactProfile {
 pub struct Artifact {
     /// The package this artifact belongs to
     pub package_id: PackageId,
+    /// Path containing the `Cargo.toml`
```

Comment:
I think it's the path to the Cargo.toml. the current doc comment sounds like it would point to the parent folder

Reply:
Yeah good catch, I'll fix it.